### DISCUSSION TITLE
Evolution/N8N bug

### DISCUSSION THREAD

**Discussion by lucirlei** [chatwoot/chatwoot#9576] [Self hosted]
When you use Evolution API to create new instance through N8N, Evolution creates a new inbox named whatsapp(xx) automatically. Then you connect the whatsapp and it is ok. Works fine. If ao go to Chatwoot and change the name of the Inbox, then chatwoot does send whatsapp new messages but stop receiving messages.

**Comment by MarcioLacerda** [comment]
Pelo que entendi a EvolutionAPI ainda não suporta mudar o nome da caixa de entrada.
https://github.com/EvolutionAPI/evolution-api/issues/192

**Comment by sojan-official** [comment]
This issue pertains to the EvolutionAPI. Please report this on the EvolutionAPI repository.
